### PR TITLE
Create Unsupported page

### LINKED ISSUE #11: Show min supported version
Currently we use these features:
- optional chain (es2020; Cr80, FF74)
- globalThis (es2020, Cr71, FF65)
- RelativeTimeFormat (Cr71, FF65)
- DateTimeFormat timeStyle, dateStyle (Cr76, FF79)

we should probably show the min supported version (80)

### PR DESCRIPTION
**Pull request by TheColaber**
Related: https://github.com/ScratchAddons/ScratchAddons/pull/2518
Resolves #11

### CODE REVIEW

File: content/unsupported.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<section class="container">
+  <p>Try reinstalling or updating your browser to run Scratch Addons</p>
+  <p>
+    <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chromium 80+ (
+    <a href="https://google.com/chrome/"><span class="iconify" data-icon="simple-icons:googlechrome"></span> Chrome 80+</a>,
+    <a href="https://microsoft.com/en-us/edge/"><span class="iconify" data-icon="simple-icons:microsoftedge"></span> Microsoft Edge 80+</a>,
+    <a href="https://opera.com/download/"><span class="iconify" data-icon="simple-icons:opera"></span> Opera 67+</a>,
+    <a href="https://brave.com/download/"><span class="iconify" data-icon="simple-icons:brave"></span> Brave 1.3+</a>,
+    <a href="https://vivaldi.com/download/"><span class="iconify" data-icon="simple-icons:vivaldi"></span> Vivaldi 2.11+</a>)
```
Comment (GrahamSH-LLK) [inline content/unsupported.html:23]: These should be site params

**Review by GrahamSH-LLK — COMMENTED** [review]
```html
---
layout: default
title: Unsupported browser version
description: Scratch Addons doesn't support this old browser version.
---

<section id="intro" class="intro-heading">
	<h1 class="mb-3">Unsupported browser version</h1>
	<p class="mb-1">Scratch Addons doesn't support this old browser version. Check out our tips below to get up and running.</p>
</section>

<section class="container">
  <p>Our minimum supported browser versions are {{ .Site.Params.min_firefox_version }} on Firefox and {{ .Site.Params.min_chrome_version }} for Chrome.</p>
    <ul>
        <li>If you're on an old version, update your browser. Read about how to do this can do this in <a href="https://support.google.com/chrome/answer/95414">Chrome</a> and <a href="https://support.mozilla.org/en-US/kb/update-firefox-latest-release">Firefox</a></li>
        <li>If Scratch Addons still doesn't work, or you get this message again, send us feedback <a href="/feedback">on our feedback page</a>.
    </ul>
</section>
```
I made this earlier. Try to include some items from it

File: content/unsupported.html (hunk 1 above)
Comment (TheColaber) [reply content/unsupported.html:23]: Please explain... W_L said he wants all links here

Comment (GrahamSH-LLK) [reply content/unsupported.html]: Use {{ .Site.Params.min_firefox_version }} and define that in the params sections of config.yml

**Comment by WorldLanguages** [comment]
Now looks like this:
image

IMO the most important part is clarifying old Chromebooks cannot be easily upgradable. I bet most of our users running old Chrome versions is because they are using chromebooks that stopped getting support.

**Comment by WorldLanguages** [comment]
@Hans5958 Do you know what the proper way of adding noindex to this page is? (so that Google won't show it as a search result)

File: content/unsupported-browser.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+<section class="container">
+  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+  <br>
+  <h4>
+    <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chromium 80+
+  </h4>
+  <p>
+    <p>
+      <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chrome 80+
+      &nbsp;<a href="https://google.com/chrome/" target="_blank">Download</a>
```
Comment (mxmou) [inline content/unsupported-browser.html:23]: I think `margin-left` (using either a class or inline CSS) is better than using `&nbsp;` for this.

File: content/unsupported-browser.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      &nbsp;&nbsp;<a href="https://support.google.com/chrome/answer/95414" target="_blank">How to update</a>
+      <br>
+      <span class="iconify" data-icon="simple-icons:microsoftedge"></span> Microsoft Edge 80+ 
+      &nbsp;<a href="https://microsoft.com/edge/" target="_blank">Download</a>
```
Comment (mxmou) [inline content/unsupported-browser.html:27]: [Link to Microsoft Support](https://support.microsoft.com/en-us/topic/microsoft-edge-update-settings-af8aaca2-1b69-4870-94fe-18822dbb7ef1) (for "How to update")

**Comment by Hans5958** [comment]
Title case already

**Comment by Hans5958** [comment]
> @Hans5958 Do you know what the proper way of adding noindex to this page is? (so that Google won't show it as a search result)

Maybe I'm going to add another front matter config for that. (Why are we hiding we can get more backlinks for that lol)

File: content/unsupported-browser.html (hunk 2 above)
Comment (Hans5958) [reply content/unsupported-browser.html:23]: Or using ul-li lists

**Review by Hans5958 — CHANGES_REQUESTED** [review]
Wow. I somehow didn't finish the review.

Title case already

**Comment by RedGuy12** [comment]
You CAN tell it “follow, noindex”

**Comment by Hans5958** [comment]
Alright, got `robots_tag` working. The front matter should look like this.

```yml
robots_tag: noindez
```

**Comment by WorldLanguages** [comment]
image

**Comment by WorldLanguages** [comment]
image

**Comment by WorldLanguages** [comment]
Would be great to know why the build testing is failing 🤔

**Comment by GrahamSH-LLK** [comment]
> Would be great to know why the build testing is failing 🤔

i18n

**Comment by RedGuy12** [comment]
Can you put `•` characters between "Download" and "How to update"?
________________________________

**Comment by Hans5958** [comment]
Apologize for the failing check. I've fixed it and now it is green as grass.

**Comment by Hans5958** [comment]
I might change it in the future to use the GitHub tables, like the ones on https://scratchaddons.com/docs/chrome-web-store-privacy-information/...

**Comment by TheColaber** [comment]
Is there any reason not to merge?

**Pull request merged**